src/test/java/seedu/address/logic/commands/AddShortcutCommandTest.java

```diff
@@ -0,0 +1,296 @@
+
+        @Override
+        public void setGuiSettings(GuiSettings guiSettings) {
+            throw new AssertionError("This method should not be called.");
+        }
+
+        @Override
+        public Path getAddressBookFilePath() {
+            throw new AssertionError("This method should not be called.");
+        }
+
+        @Override
+        public void setAddressBookFilePath(Path addressBookFilePath) {
+            throw new AssertionError("This method should not be called.");
+        }
+
+        @Override
+        public Path getShortcutLibraryFilePath() {
+            return null;
+        }
+
+        @Override
+        public void setShortcutLibraryFilePath(Path shortcutLibraryFilePath) {
+
+        }
+
+        @Override
+        public void addPerson(Person person) {
+            throw new AssertionError("This method should not be called.");
+        }
+
+        @Override
+        public void setAddressBook(ReadOnlyAddressBook newData) {
+            throw new AssertionError("This method should not be called.");
+        }
+
+        @Override
+        public ReadOnlyAddressBook getAddressBook() {
+            throw new AssertionError("This method should not be called.");
+        }
+
+        @Override
+        public boolean hasPerson(Person person) {
+            throw new AssertionError("This method should not be called.");
+        }
+
+        @Override
+        public void deletePerson(Person target) {
+            throw new AssertionError("This method should not be called.");
+        }
+
+        @Override
+        public void setPerson(Person target, Person editedPerson) {
+            throw new AssertionError("This method should not be called.");
+        }
+
+        @Override
+        public ObservableList<Person> getFilteredPersonList() {
+            throw new AssertionError("This method should not be called.");
+        }
+
+        @Override
+        public void updateFilteredPersonList(Predicate<Person> predicate) {
+            throw new AssertionError("This method should not be called.");
+        }
+
+        @Override
+        public void updatePersonListByAttribute(List<Attribute> attributeTypes) {
+            throw new AssertionError("This method should not be called.");
+        }
+
+        @Override
+        public Authentication getAuthentication() {
+            throw new AssertionError("This method should not be called.");
+        }
+
+        @Override
+        public void undoListModification() {
+            throw new AssertionError("This method should not be called.");
+        }
+
+        @Override
+        public void updateSortedPersonList(Comparator<Person> comparator) {
+            throw new AssertionError("This method should not be called.");
+        }
+
+        @Override
+        public ObservableList<Person> getWholePersonList() {
+            return null;
+        }
+
+        @Override
+        public void setShortcutLibrary(ShortcutLibrary shortcutLibrary) {
+
+        }
```

Comment:
Another empty method where you might want to throw AssertionError if they are not supposed to be called.